daffodil-test/src/test/resources/org/apache/daffodil/section17/calc_value_properties/AQ.tdml

```diff
@@ -76,46 +76,4 @@
     </tdml:infoset>
   </tdml:parserTestCase>
 
-  <tdml:defineSchema name="AQ_neg.dfdl.xsd">
-    <xs:include schemaLocation="org/apache/daffodil/xsd/DFDLGeneralFormat.dfdl.xsd"/>
-    <dfdl:format ref="ex:GeneralFormat" separator=""
-      initiator="" terminator="" emptyValueDelimiterPolicy="none"
-      textNumberRep="standard" representation="text"
-      occursCountKind="stopValue" lengthKind="delimited" lengthUnits="bytes"
-      encoding="US-ASCII" ignoreCase="no" />
-    <!-- Basic Math in DFDL Representations -->
-    <xs:element name="ROOT">
-      <xs:complexType>
-        <xs:sequence>
-          <xs:element name="x" type="xsd:float"
-            dfdl:terminator="," dfdl:encoding="US-ASCII" />
-          <xs:element name="y" type="xsd:float"
-            dfdl:encoding="US-ASCII" dfdl:terminator="%NL;%WSP*;" />
-          <xs:element name="Multiply" type="xsd:float"
-            dfdl:inputValueCalc="{ ../tns:x * ../tns:y }" />
-          <xs:element name="Add" type="xsd:float"
-            dfdl:inputValueCalc="{ ../tns:x + ../tns:y }" />
-          <xs:element name="Subtract" type="xsd:float"
-            dfdl:inputValueCalc="{ ../tns:x - ../tns:y }" />
-          <xs:element name="SubtractLiteral" type="xsd:float"
-            dfdl:inputValueCalc="{ ../tns:x - 2 }" />
-        </xs:sequence>
-      </xs:complexType>
-    </xs:element>
-  </tdml:defineSchema>
-
-  <tdml:parserTestCase name="AQ001" root="ROOT"
-    model="AQ_neg.dfdl.xsd"
-    description="Section 17 Tests variables, arithmetic, and inputValueCalc - DFDL-17-007R">
-
-    <tdml:document>
-      <tdml:documentPart type="text"><![CDATA[10.5,12.01
- ]]></tdml:documentPart>
-    </tdml:document>
-
-    <tdml:errors>
-      <tdml:error>Schema definition error</tdml:error>
-    </tdml:errors>
-
-  </tdml:parserTestCase>
```

Comment:
Schema and test completely removed. Was that intentional?

Reply:
Yes, the schema and input were both identical to AQ000.  I'm not sure why AQ001 was expected to fail, but my guess is that someone copied AQ000 to set up a negative test case, but never changed it.